_schema.graphql

```diff
@@ -4560,9 +4578,10 @@ type HomePageModuleContextFair {
   hours: String
   href: String
   image: Image
+  is_active: Boolean @deprecated(reason: "Prefer isActive instead")
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 4581;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `is_active`